raw/map.go

```diff
@@ -13,6 +14,25 @@ import (
 // StringSliceMap represents an array of strings or a map of strings.
 type StringSliceMap map[string]string
 
+// Value returns the map in JSON format
+func (s StringSliceMap) Value() (driver.Value, error) {
+	value, err := json.Marshal(s)
+	if err != nil {
+		return nil, err
+	}
+	return string(value), nil
+}
+
+// Scan decodes the JSON string into map[string]string
```

Comment:
Hate to be nit-picky but could we get `.` to follow this comment?

```suggestion
// Scan decodes the JSON string into map[string]string.
```